Gemfile.lock

```diff
@@ -17,6 +17,7 @@ GEM
     ethon (0.9.1)
       ffi (>= 1.3.0)
     ffi (1.9.14)
+    ffi (1.9.14-x64-mingw32)
```

Comment:
Not needing to commit these specific changes is all I meant.  Off the top of my head, I'm not sure if committing Gemfile.lock is necessary for the gh-pages deployment - I assume it is.  I'll look into this and see if we could .gitignore it.